includes/functions.php

```diff
@@ -410,3 +410,29 @@ function indieauth_get_me() {
 	}
 	return $response['me'];
 }
+
+/* Returns jf2 formatted user data
+ *
+ */
+function indieauth_get_user( $user ) {
+	if ( is_numeric( $user ) ) {
+		$user = get_user_by( 'ID', $user );
+	}
+	if ( ! $user instanceof WP_User ) {
+		return array();
+	}
+	$return = array(
+		'type'  => 'card',
+		'name'  => $user->display_name,
+		'url'   => empty( $user->user_url ) ? get_author_posts_url( $user->ID ) : $user->user_url,
```

Comment:
shouldn't we always use the `author_posts_url`? Because the `user_url` might be a completely different domain.

Reply:
Remember, this is a profile return. Not the 'me' return which is part of the client_id process, and shouldn't change. This is what the person wants to share as their site.